sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSiteVersionPolicyJobProgress.md

```diff
@@ -0,0 +1,66 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spositeversionpolicyjobprogress
+applicable: SharePoint Online
+title: Get-SPOSiteVersionPolicyJobProgress
+schema: 2.0.0
+author: msjennywu
+ms.author: jennywu
+ms.reviewer:
+manager: seanmc
+---
+
+# Get-SPOSiteVersionPolicyJobProgress
+
+## SYNOPSIS
+
+> [!NOTE]
+> This feature is part of the version history controls preview. If your tenant is not part of the preview or the feature has not rolled out to your tenant, you will get an error when trying to run this cmdlet.
+
+Gets the progress of setting version policy for existing document libraries on the site collection.
```

Comment:
Tracks progress of version history limit update on existing document libraries on the site collection.